project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16
```

Comment:
Not using 1.0.x because of https://github.com/tototoshi/sbt-slick-codegen/pull/14 (merged but not released).